source/frontend/src/features/mapSideBar/acquisition/AcquisitionView.tsx

```diff
@@ -76,9 +76,6 @@ export const AcquisitionView: React.FunctionComponent<IAcquisitionViewProps> = (
   const history = useHistory();
   const match = useRouteMatch();
   const { file, lastUpdatedBy } = useContext(SideBarContext);
-  if (!!file && file?.fileType !== FileTypes.Acquisition) {
-    throw Error('Context file is not an acquisition file');
-  }
```

Comment:
I actually think this is a good error check. Without it we would have never found this bug/race condition. Just my opinion, but I would actually like to see this restored and a similar check added to ResearchView - but we can discuss on our Friday meeting

Reply:
If we leave it in, it would be dead code, because the parent makes the same check. This if statement will never run with the recommended changes.

Reply:
well, in theory this view could be embedded "anywhere" by another container but as I said would like us to discuss this use case for other views that make use of a context that might not have the information the view is expecting